packages/extension-chakra-storefront/config/default.json

```diff
@@ -1,50 +1,144 @@
 {
     "enabled": true,
     "activeDataEnabled": false,
+    "maxCacheAge": 900,
+    "staleWhileRevalidate": 900,
     "commerceAPI": {
         "proxyPath": "/mobify/proxy/api",
         "parameters": {
-            "clientId": "<CLIENT_ID>",
-            "organizationId": "<ORGANIZATION_ID>",
-            "shortCode": "<SHORT_CODE>",
-            "siteId": "<SITE_ID>"
+            "clientId": "c9c45bfd-0ed3-4aa2-9971-40f88962b836",
```

Comment:
TODO: revert before merging

Reply:
Yup, if there are required properties, we can use placeholders for them.